### PR TITLE
Manage Hyphens with CSS

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/2016/10/css-hyphens.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Chrome 55 impelements the hyphens property to control when soft hyphens appear and how they behave."
+
+{# wf_updated_on: 2016-10-21 #}
+{# wf_published_on: 2016-10-21 #}
+{# wf_tags: chrome55,css, #}
+{# wf_featured_snippet: Chrome 55 impelements the hyphens property to control when soft hyphens appear and how they behave. #}
```
Comment (petele) [inline src/content/en/updates/2016/10/css-hyphens.md]: Updates need to have a wf_featured_image

File: src/content/en/updates/2016/10/css-hyphens.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+A soft hyphen is one that will only be shown when it occurs at the trailing
+margin. How this hyphen renders in Chrome 55 or later depends on the value of
+the CSS `hypens` property.
+
+    -webkit-hyphens: manual;
+    hyphens: manual;
+    
+Combining these gives a result like this:
+
+<img src="/web/updates/images/2016/10/css-hyphen/single-line.png" width="">
```
Comment (petele) [inline src/content/en/updates/2016/10/css-hyphens.md]: Remove the width param

File: src/content/en/updates/2016/10/css-hyphens.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+{% framebox height="24px" %}
+<style>
+div {
+  font: 18px serif;
+  margin-bottom: 2.5%;
+  background-color: #e1f5fe;
+}
+
+div.none {
```
Comment (petele) [inline src/content/en/updates/2016/10/css-hyphens.md]: please rename this `div.manual` and set the class accordingly

File: src/content/en/updates/2016/10/css-hyphens.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
```
Comment (petele) [inline src/content/en/updates/2016/10/css-hyphens.md]: please rename this `div.auto` and set the class accordingly

File: src/content/en/updates/2016/10/css-hyphens.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+   -webkit-hyphens: auto;
+   hyphens: auto;
+}
+</style>
+<div class="none">
+  Google ipsum dolor sit amet, consectetur adipiscing elit.
+</div>
+{% endframebox %}
+
++{% include "comment-widget.html" %}
```
Comment (petele) [inline src/content/en/updates/2016/10/css-hyphens.md]: Remove the extra plus sign

**Comment by jpmedley** [comment]
Changes made.

File: src/content/en/updates/2016/10/css-hyphens.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Chrome 55 impelements the hyphens property to control when soft hyphens appear and how they behave."
+
+{# wf_updated_on: 2016-10-21 #}
+{# wf_published_on: 2016-10-21 #}
+{# wf_tags: chrome55,css, #}
```
Comment (petele) [inline src/content/en/updates/2016/10/css-hyphens.md]: one more - remove the extra comma at the end of the file

**Comment by marccantin** [comment]
STOP YOUR SPAM!!!!!!!!!!!!!!!!

2016-10-21 16:34 GMT-04:00 Joe Medley <notifications@github.com>:

**Comment by petele** [comment]
@marccantin - Sorry, you've likely subscribed to notification on this GitHub repo (https://github.com/google/WebFundamentals/). Please check your notifications on GitHub and unsubscribe there.

**Pull request merged**